### PR TITLE
Test that transforms print > Transforming

### PR DESCRIPTION
**Pull request by wolfs**
since those messages will be filtered from the build scan logs.

### CODE REVIEW

File: subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/TransformationLoggingIntegrationTest.groovy [hunk 1]
```diff
@@ -172,6 +172,56 @@ class TransformationLoggingIntegrationTest extends AbstractConsoleGroupedTaskFun
...
+                RedMultiplier() {
+                    super("red")
+                }
+                
+                @Override
+                void transform(TransformOutputs outputs) {
+                    // BlockingHttpServer seems to be too verbose, so blocking here causes
+                    // the progress message to disappear somehow.
+                    // Thread.sleep seems to work just fine.
+                    Thread.sleep(200)
```
Comment (lptr) [inline subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/TransformationLoggingIntegrationTest.groovy]: Can we fix this problem somehow?

File: subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformationNode.java [hunk 2]
```diff
@@ -245,13 +246,16 @@ public void resolveDependencies(TaskDependencyResolver dependencyResolver, Actio
...
+        private static final String TRANSFORMING_PROGRESS_PREFIX = "Transforming ";
+
         @Override
         public final BuildOperationDescriptor.Builder description() {
             String transformerName = transformationStep.getDisplayName();
             String subjectName = describeSubject();
             String basicName = subjectName + " with " + transformerName;
             return BuildOperationDescriptor.displayName("Transform " + basicName)
-                .progressDisplayName("Transforming " + basicName)
+                .progressDisplayName(TRANSFORMING_PROGRESS_PREFIX + basicName)
```
Comment (lptr) [inline subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformationNode.java]: Would it make sense to concatenate `TRANSFORMING_PROGRESS_PREFIX` to `basicName`? I think that way we could remove one allocation.

File: subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/TransformationLoggingIntegrationTest.groovy (hunk 1 above)
Comment (wolfs) [reply subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/TransformationLoggingIntegrationTest.groovy]: I tried and I couldn't. Do you have an idea how?

Comment (lptr) [reply subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/TransformationLoggingIntegrationTest.groovy]: I don't have an idea, but using `Thread.sleep()` to work around the problem seems fishy. Your call.

**Pull request merged**